doc/api/stream.md

```diff
@@ -548,6 +548,15 @@ added: v9.3.0
 Return the value of `highWaterMark` passed when constructing this
 `Writable`.
 
+##### `writable.writableHighWaterMark`
```

Comment:
I think this should just be merged into the existing description, otherwise it looks like a documentation error with multiple definitions.

Reply:
Yep, makes sense. I am not sure on how to merge the.   any ideas?